primeqa/mrc/README.md

```diff
@@ -249,13 +249,61 @@ Additionally, to specify a MRQA subset e.g. `SQuAD`, `NaturalQuestionsShort`, `T
        --dataset_filter_column_name subset
 ```
 
-Cross domain experiments can be run by running train and eval as separate processes. The
+Cross domain experiments can be run by running train and eval as separate processes.
 
  
 ### Custom Data
 
 Users can also train (fine-tune) and evaluate the MRC model on custom data by providing their own train_file and eval_file. Instructions for getting started are available [here](../../examples/custom_mrc/README.md).
 
+
+### Training with Multiple Datasets
+
+PrimeQA supports the training of MRC model with combination of multiple datasets, which are specified in "--train_fof" argument. Three formats, csv, jsonl and json, can be used to define "--train_file".
+
+In csv format, the columns of each line, separated by space, are for:
```

Comment:
"Each line of `train_fof` consists of four columns, separated by spaces:"

(Don't say `csv` because the automatic expectation in that case is that the columns are comma-separated.)

Reply:
@ma-sultan Fixed.